qutebrowser/misc/earlyinit.py

```diff
@@ -186,6 +187,7 @@ def check_ssl_support():
         from PyQt5.QtNetwork import QSslSocket
     except ImportError:
         _die("Fatal error: Your Qt is built without SSL support.")
+    # pylint: enable=unused-variable
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.